packages/dev/docs/pages/blog/drag-and-drop.mdx

```diff
@@ -0,0 +1,243 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import dndMobile from 'url:../assets/dnd-mobile.mp4';
+import dndMobileVTT from 'url:../assets/dnd-mobile.vtt';
+import dndKeyboard from 'url:../assets/dnd-keyboard.mp4';
+import heroVideo from 'url:../assets/dnd.mp4';
+import BetweenDropPosition from '@react-aria/dnd/docs/BetweenDropPosition.svg';
+import OnDropPosition from '@react-aria/dnd/docs/OnDropPosition.svg';
+import RootDropPosition from '@react-aria/dnd/docs/RootDropPosition.svg';
+
+import {BlogPostLayout, Video, Track, Image} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+keywords: [drag and drop, dnd, components, accessibility, keyboard, mobile, react spectrum, react, spectrum, interactions, touch]
+description: We are excited to announce the release of drag and drop support in [React Aria](https://react-spectrum.adobe.com/react-aria/dnd.html) and [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/dnd.html)! This includes a suite of hooks for implementing drag and drop interactions, with support for both mouse and touch, as well as full parity for keyboard and screen reader input.
+date: 2022-11-10
+author: '[Devon Govett](https://twitter.com/devongovett)'
+---
+
+# Drag and drop
```

Comment:
todo: better title

Reply:
something something dragons :). "Slaying dragons (drop)"

more boring: "Introducing keyboard and screenreader drag and drop" or something